addons/website_sale_picking/models/sale_order.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
```

Comment:
```suggestion
    _inherit = 'sale.order'
```
nitpicking for future PRs (or for the forward-ports :D).